network-config.js

```diff
@@ -43,10 +43,10 @@ const networkConfig = {
   },
   sepolia: {
     linkToken: "0x779877A7B0D9E8603169DdbD7836e478b4624789",
-    linkEthPriceFeed: "",
-    functionsOracle: "0x88F9Dd856D2c8CeC68569b57858ee953CC2bB4f4",
-    functionsOracleFactory: "0x85b44e2A2812E0907136357bb5F536da7cC99c4A",
-    functionsOracleRegistry: "0x774Ea40eCC9fBC7861b35d57B5f49E13Ce87d75C",
+    linkEthPriceFeed: "0x42585eD362B3f1BCa95c640FdFf35Ef899212734",
+    functionsOracle: "0x0019f8318006EB71594c4a6CF3cfcf1B25b2174b",
```

Comment:
This needs to happen separately, when we update the DON.

Reply:
I do agree this change is unrelated to the topic of this PR. Let's revert this part and everything else looks good.